lib/webmock/http_lib_adapters/em_http_request_adapter.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+return if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.4.0')
```

Comment:
Could we have something like:

```
unless ENV["IGNORE_RUBY_VERSION_CHECK_FOR_EM"] == "enabled"
  return if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.4.0')
end
```
We've forked em-http-request where I work, so this isn't a problem.